Sources/ViewInspector/BaseTypes.swift

```diff
@@ -9,33 +81,17 @@ public protocol InspectableProtocol {
 
 @available(iOS 13.0, macOS 10.15, tvOS 13.0, *)
 public extension InspectableProtocol where Self: View {
-    
+
     func extractContent(environmentObjects: [AnyObject]) throws -> Any {
-        var copy = self
-        environmentObjects.forEach { copy = EnvironmentInjection.inject(environmentObject: $0, into: copy) }
-        let missingObjects = EnvironmentInjection.missingEnvironmentObjects(for: copy)
-        if missingObjects.count > 0 {
-            let view = Inspector.typeName(value: self)
-            throw InspectionError
-                .missingEnvironmentObjects(view: view, objects: missingObjects)
-        }
-        return copy.body
+        try Inspectable.withView(self).extractContent(environmentObjects: environmentObjects)
```

Comment:
Eventually we could inline this code and remove the `extractContent(…)` method from `InspectableProtocol`, which would get us closer to relaxing the requirement that views conform to `InspectableProtocol` in order to be inspected.